build-farm/platform-specific-configurations/linux.sh

```diff
@@ -68,6 +68,9 @@ then
     then
       export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --enable-cuda --with-cuda=$CUDA_HOME"
     fi
+  elif [ "${ARCHITECTURE}" == "aarch64" ] && [ -f /usr/local/cuda-10.2/include/cuda.h ]; then
```

Comment:
@sxa is this change meant to be part of this PR?

Reply:
No - I accidentally made a change in my master branch so all my PRs got based from that one ... I'll fix

Reply:
Done